schemas/v1.0/schema.yaml

```diff
@@ -0,0 +1,525 @@
+    $ref: '#/$defs/specification-extensions'
+    unevaluatedProperties: false
+  failure-action-object:
+    $comment: https://spec.openapis.org/arazzo/v1.0.0#failure-action-object
+    description: A single failure action which describes an action to take upon failure of a workflow step
+    type: object
+    properties:
+      name:
+        description: The name of the failure action
+        type: string
+      type:
+        description: The type of action to take
+        enum:
+          - end
+          - goto
+          - retry
+      workflowId:
+        description: The workflowId referencing an existing workflow within the Arazzo description to transfer to upon failure of the step
+        $dynamicRef: '#workflowId'
+      stepId:
+        description: The stepId to transfer to upon failure of the step
+        $dynamicRef: '#stepId'
+      retryAfter:
+        description: A non-negative decimal indicating the seconds to delay after the step failure before another attempt SHALL be made
+        type: number
+        minimum: 0
+      retryLimit:
+        description: A non-negative integer indicating how many attempts to retry the step MAY be attempted before failing the overall step
+        type: integer
+        minimum: 0
+      criteria:
+        description: A list of assertions to determine if this action SHALL be executed
+        type: array
+        uniqueItems: true
+        items:
+          $ref: '#/$defs/criterion-object'
+    allOf:
+      - if:
+          properties:
+            type:
+              enum:
+                - goto
+                - retry
+        then:
+          oneOf:
+            - required:
+                - workflowId
+            - required:
+                - stepId
+      - if:
+          properties:
+            type:
+              const: retry
+        then:
+          required:
+            - retryAfter
+    required:
+      - name
+      - type
+    $ref: '#/$defs/specification-extensions'
+    unevaluatedProperties: false
+  reusable-object:
+    $comment: https://spec.openapis.org/arazzo/v1.0.0#reusable-object
+    description: A simple object to allow referencing of objects contained within the Components Object
+    type: object
+    properties:
+      reference:
+        description: A runtime expression used to reference the desired object
+        type: string
+      value:
+        description: Sets a value of the referenced parameter
+        type: string
+    required:
+      - reference
+    unevaluatedProperties: false
+  parameter-object:
+    description: Describes a single step parameter
+    type: object
+    properties:
+      name:
+        description: The name of the parameter
+        type: string
+      in:
+        description: The named location of the parameter
+        enum:
+          - path
+          - query
+          - header
+          - cookie
+          - body
+      value:
+        description: The value to pass in the parameter
+        type: string
```

Comment:
not sure that this must be a string , this matter especially if we handle json , is it a must to have " 

for instance look over there 
https://github.com/OAI/Arazzo-Specification/blob/main/examples/1.0.0/pet-coupons.arazzo.yaml#L74

Reply:
that's fair for a Parameter object.  I updated it to all types. 

The example you provided is invalid because a `reusableObject>value` can only be a string.

Reply:
although, you can argue that the text indicates the `value` is only for `Parameter Object` references, which indeed takes ANY value.  So we need the spec maintainers to review that behavior